src/js/feedback/components/FeedbackSubmitted.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function FeedbackSubmitted({ shouldSendResponse }) {
+  return (
+    <div id="feedback-submitted">
+      <h4 className="feedback-widget-title">Thank you for your feedback!</h4>
+      {shouldSendResponse ? (
```

Comment:
Ignore me if you know about this and just prefer the ternary syntax, but this is slightly more concise: `{shouldSendResponse && <p className="feedback-widget-intro">We'll get back to you soon.</p>}`

Reply:
I swear there was a version of React where that printed `false` in the rendered HTML lol